theme/templates/resource-landing-page/references.html

```diff
@@ -1,7 +1,7 @@
 {% load hydroshare_tags crispy_forms_tags %}
 
 {% if not resource_edit_mode %}
-    {% if sources or show_relations_section %}
+    {% if sources or show_relations_section or belongs_to_collections %}
```

Comment:
@kjlippold Why this ```belongs_to_collections``` condition is added here?

Reply:
I added this condition in response to https://github.com/hydroshare/hydroshare/pull/4023#issuecomment-724189023. The 'References' major heading wasn't appearing in collections, so there wasn't a break between References and the preceding section. I'm not sure if this is intentional or not, but if it isn't it should probably be resolved separately.